pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -97,26 +96,23 @@ func (r *ReconcileJaeger) Reconcile(request reconcile.Request) (reconcile.Result
 
 	// wait for all the dependencies to succeed
 	if err := r.handleDependencies(str); err != nil {
+		logFields.WithError(err).Error("failed to handle the dependencies")
 		return reconcile.Result{}, err
 	}
 
-	created, err := r.handleCreate(str)
+	applied, err := r.apply(*instance, str)
 	if err != nil {
-		log.WithField("instance", instance).WithError(err).Error("failed to create")
+		logFields.WithError(err).Error("failed to apply the changes")
```

Comment:
The error is returned below, Is it also being logged upstream? You might also wrap the error before returning.

Reply:
This is the "entry point" for our application. It might be logged before, but this is the closed to the user as we have inside the operator.